### ISSUE TITLE
  package ‘rstanram’ is not available (for R version 3.3.2)

### ISSUE THREAD

**Issue by gmichel-SMU** [stan-dev/rstanarm#215]
#### Summary:
I'm trying to install rstanram on IBM DataScience portal (Could service) but using github installation or install_package() bot are yielding errors.
#### Description:
Issuing the command install.packages("rstanarm")
produce the following error:

package ‘rstanram’ is not available (for R version 3.3.2)

Using GitHub installation, end with the following error:

> library(devtools) 
> install_github("stan-dev/rstanarm", args = "--preclean", build_vignettes = FALSE) Downloading GitHub repo stan-dev/rstanarm@master from URL https://api.github.com/repos/stan-dev/rstanarm/zipball/master Installing rstanarm trying URL 'https://cran.rstudio.com/src/contrib/bayesplot_1.2.0.tar.gz' Content type 'unknown' length 4198517 bytes (4.0 MB) ================================================== downloaded 4.0 MB  Installing bayesplot '/usr/lib64/R/bin/R' --no-site-file --no-environ --no-save --no-restore --quiet  \   CMD INSTALL '/tmp/Rtmp8XJn7C/devtools197f28390d/bayesplot'  \   --library='/home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3' --install-tests   * installing *source* package ‘bayesplot’ ... ** package ‘bayesplot’ successfully unpacked and MD5 sums checked ** R ** inst ** tests ** preparing package for lazy loading ** help *** installing help indices Warning in file.copy(file.path(R.home("doc"), "html", "R.css"), outman) :   problem copying /usr/share/doc/R-3.3.2/html/R.css to /home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3/bayesplot/html/R.css: No such file or directory   converting help for package ‘bayesplot’     finding HTML links ... done     MCMC-combos                             html       MCMC-diagnostics                        html   Rd warning: /tmp/Rtmp8XJn7C/devtools197f28390d/bayesplot/man/MCMC-diagnostics.Rd:34: missing file link ‘geom_line’     MCMC-distributions                      html       MCMC-intervals                          html       MCMC-nuts                               html       MCMC-overview                           html       MCMC-recover                            html       MCMC-scatterplots                       html       MCMC-traces                             html       PPC-discrete                            html   Rd warning: /tmp/Rtmp8XJn7C/devtools197f28390d/bayesplot/man/PPC-discrete.Rd:42: missing file link ‘geom_pointrange’ Rd warning: /tmp/Rtmp8XJn7C/devtools197f28390d/bayesplot/man/PPC-discrete.Rd:44: missing file link ‘geom_line’     PPC-distributions                       html       PPC-errors                              html       PPC-intervals                           html   Rd warning: /tmp/Rtmp8XJn7C/devtools197f28390d/bayesplot/man/PPC-intervals.Rd:52: missing file link ‘geom_pointrange’     PPC-loo                                 html   Rd warning: /tmp/Rtmp8XJn7C/devtools197f28390d/bayesplot/man/PPC-loo.Rd:52: missing file link ‘geom_pointrange’     PPC-overview                            html       PPC-scatterplots                        html       PPC-test-statistics                     html       available_ppc                           html       bayesplot-colors                        html       bayesplot-extractors                    html       bayesplot-helpers                       html   Rd warning: /tmp/Rtmp8XJn7C/devtools197f28390d/bayesplot/man/bayesplot-helpers.Rd:79: missing file link ‘geom_vline’ Rd warning: /tmp/Rtmp8XJn7C/devtools197f28390d/bayesplot/man/bayesplot-helpers.Rd:80: missing file link ‘geom_hline’ Rd warning: /tmp/Rtmp8XJn7C/devtools197f28390d/bayesplot/man/bayesplot-helpers.Rd:84: missing file link ‘element_rect’ Rd warning: /tmp/Rtmp8XJn7C/devtools197f28390d/bayesplot/man/bayesplot-helpers.Rd:87: missing file link ‘element_text’ Rd warning: /tmp/Rtmp8XJn7C/devtools197f28390d/bayesplot/man/bayesplot-helpers.Rd:90: missing file link ‘element_line’ Rd warning: /tmp/Rtmp8XJn7C/devtools197f28390d/bayesplot/man/bayesplot-helpers.Rd:96: missing file link ‘geom_vline’ Rd warning: /tmp/Rtmp8XJn7C/devtools197f28390d/bayesplot/man/bayesplot-helpers.Rd:110: missing file link ‘element_blank’ Rd warning: /tmp/Rtmp8XJn7C/devtools197f28390d/bayesplot/man/bayesplot-helpers.Rd:116: missing file link ‘element_line’     bayesplot-package                       html       bayesplot_grid                          html       example-data                            html       pp_check                                html       theme_default                           html        finding level-2 HTML links ... done  Rd warning: /tmp/Rtmp8XJn7C/devtools197f28390d/bayesplot/man/theme_default.Rd:12: missing file link ‘theme_bw’ *** copying figures ** building package indices ** installing vignettes ** testing if installed package can be loaded * DONE (bayesplot) trying URL 'https://cran.rstudio.com/src/contrib/Rcpp_0.12.12.tar.gz' Content type 'unknown' length 2421289 bytes (2.3 MB) ================================================== downloaded 2.3 MB  Installing Rcpp '/usr/lib64/R/bin/R' --no-site-file --no-environ --no-save --no-restore --quiet  \   CMD INSTALL '/tmp/Rtmp8XJn7C/devtools194bd32ab5/Rcpp'  \   --library='/home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3' --install-tests   * installing *source* package ‘Rcpp’ ... ** package ‘Rcpp’ successfully unpacked and MD5 sums checked ** libs g++ -m64 -I/usr/include/R -DNDEBUG -I../inst/include/ -I/usr/local/include    -fpic  -O3 -mtune=native -march=native -Wno-unused-variable -Wno-unused-function -c Date.cpp -o Date.o g++ -m64 -I/usr/include/R -DNDEBUG -I../inst/include/ -I/usr/local/include    -fpic  -O3 -mtune=native -march=native -Wno-unused-variable -Wno-unused-function -c Module.cpp -o Module.o g++ -m64 -I/usr/include/R -DNDEBUG -I../inst/include/ -I/usr/local/include    -fpic  -O3 -mtune=native -march=native -Wno-unused-variable -Wno-unused-function -c Rcpp_init.cpp -o Rcpp_init.o g++ -m64 -I/usr/include/R -DNDEBUG -I../inst/include/ -I/usr/local/include    -fpic  -O3 -mtune=native -march=native -Wno-unused-variable -Wno-unused-function -c api.cpp -o api.o g++ -m64 -I/usr/include/R -DNDEBUG -I../inst/include/ -I/usr/local/include    -fpic  -O3 -mtune=native -march=native -Wno-unused-variable -Wno-unused-function -c attributes.cpp -o attributes.o g++ -m64 -I/usr/include/R -DNDEBUG -I../inst/include/ -I/usr/local/include    -fpic  -O3 -mtune=native -march=native -Wno-unused-variable -Wno-unused-function -c barrier.cpp -o barrier.o g++ -m64 -shared -L/usr/lib64/R/lib -Wl,-z,relro -o Rcpp.so Date.o Module.o Rcpp_init.o api.o attributes.o barrier.o -L/home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3/StanHeaders/lib -lStanHeaders -L/usr/lib64/R/lib -lR installing to /home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3/Rcpp/libs ** R ** inst ** tests ** preparing package for lazy loading ** help *** installing help indices Warning in file.copy(file.path(R.home("doc"), "html", "R.css"), outman) :   problem copying /usr/share/doc/R-3.3.2/html/R.css to /home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3/Rcpp/html/R.css: No such file or directory   converting help for package ‘Rcpp’     finding HTML links ... done     CppClass-class                          html       CppConstructor-class                    html       CppField-class                          html       CppFunction-class                       html       CppObject-class                         html       CppOverloadedMethods-class              html       DollarNames-methods                     html       Module-class                            html       Module                                  html       Rcpp-deprecated                         html       Rcpp-internal                           html       Rcpp-package                            html       Rcpp.package.skeleton                   html       Rcpp.plugin.maker                       html       RcppLdFlags                             html       RcppUnitTests                           html       compileAttributes                       html       compilerCheck                           html       cppFunction                             html       demangle                                html       dependsAttribute                        html       evalCpp                                 html       exportAttribute                         html       exposeClass                             html       formals                                 html       interfacesAttribute                     html       loadModule                              html       loadRcppModules-deprecated              html       pluginsAttribute                        html       populate                                html       registerPlugin                          html       setRcppClass                            html       sourceCpp                               html   ** building package indices ** installing vignettes ** testing if installed package can be loaded * DONE (Rcpp) trying URL 'https://cran.rstudio.com/src/contrib/rstantools_1.3.0.tar.gz' Content type 'unknown' length 44959 bytes (43 KB) ================================================== downloaded 43 KB  Installing rstantools '/usr/lib64/R/bin/R' --no-site-file --no-environ --no-save --no-restore --quiet  \   CMD INSTALL '/tmp/Rtmp8XJn7C/devtools1963720956/rstantools'  \   --library='/home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3' --install-tests   * installing *source* package ‘rstantools’ ... ** package ‘rstantools’ successfully unpacked and MD5 sums checked ** R ** inst ** tests ** preparing package for lazy loading ** help *** installing help indices Warning in file.copy(file.path(R.home("doc"), "html", "R.css"), outman) :   problem copying /usr/share/doc/R-3.3.2/html/R.css to /home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3/rstantools/html/R.css: No such file or directory   converting help for package ‘rstantools’     finding HTML links ... done     bayes_R2                                html       init_cpp                                html       log_lik                                 html       loo-prediction                          html       posterior_interval                      html       posterior_linpred                       html       posterior_predict                       html       predictive_error                        html       predictive_interval                     html       prior_summary                           html       rstan_package_skeleton                  html       rstantools-package                      html   *** copying figures ** building package indices ** installing vignettes ** testing if installed package can be loaded * DONE (rstantools) trying URL 'https://cran.rstudio.com/src/contrib/shinystan_2.4.0.tar.gz' Content type 'unknown' length 2394490 bytes (2.3 MB) ================================================== downloaded 2.3 MB  Installing shinystan trying URL 'https://cran.rstudio.com/src/contrib/colourpicker_0.3.tar.gz' Content type 'unknown' length 1856742 bytes (1.8 MB) ================================================== downloaded 1.8 MB  Installing colourpicker trying URL 'https://cran.rstudio.com/src/contrib/miniUI_0.1.1.tar.gz' Content type 'unknown' length 7787 bytes ================================================== downloaded 7787 bytes  Installing miniUI '/usr/lib64/R/bin/R' --no-site-file --no-environ --no-save --no-restore --quiet  \   CMD INSTALL '/tmp/Rtmp8XJn7C/devtools19142c37b9/miniUI'  \   --library='/home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3' --install-tests   * installing *source* package ‘miniUI’ ... ** package ‘miniUI’ successfully unpacked and MD5 sums checked ** R ** inst ** preparing package for lazy loading ** help *** installing help indices Warning in file.copy(file.path(R.home("doc"), "html", "R.css"), outman) :   problem copying /usr/share/doc/R-3.3.2/html/R.css to /home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3/miniUI/html/R.css: No such file or directory   converting help for package ‘miniUI’     finding HTML links ... done     miniButtonBlock                         html        finding level-2 HTML links ... done      miniContentPanel                        html   Rd warning: /tmp/Rtmp8XJn7C/devtools19142c37b9/miniUI/man/miniContentPanel.Rd:13: missing file link ‘fillCol’     miniPage                                html       miniTabstripPanel                       html       miniTitleBar                            html   ** building package indices ** testing if installed package can be loaded * DONE (miniUI) trying URL 'https://cran.rstudio.com/src/contrib/shinyjs_0.9.1.tar.gz' Content type 'unknown' length 977767 bytes (954 KB) ================================================== downloaded 954 KB  Installing shinyjs '/usr/lib64/R/bin/R' --no-site-file --no-environ --no-save --no-restore --quiet  \   CMD INSTALL '/tmp/Rtmp8XJn7C/devtools1950f61971/shinyjs'  \   --library='/home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3' --install-tests   * installing *source* package ‘shinyjs’ ... ** package ‘shinyjs’ successfully unpacked and MD5 sums checked ** R ** inst ** tests ** preparing package for lazy loading ** help *** installing help indices Warning in file.copy(file.path(R.home("doc"), "html", "R.css"), outman) :   problem copying /usr/share/doc/R-3.3.2/html/R.css to /home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3/shinyjs/html/R.css: No such file or directory   converting help for package ‘shinyjs’     finding HTML links ... done     classFuncs                              html       colourInput                             html       colourPicker                            html       delay                                   html       disabled                                html   Rd warning: /tmp/Rtmp8XJn7C/devtools1950f61971/shinyjs/man/disabled.Rd:48: missing file link ‘toggleState’ Rd warning: /tmp/Rtmp8XJn7C/devtools1950f61971/shinyjs/man/disabled.Rd:49: missing file link ‘enable’ Rd warning: /tmp/Rtmp8XJn7C/devtools1950f61971/shinyjs/man/disabled.Rd:50: missing file link ‘disable’     extendShinyjs                           html       hidden                                  html   Rd warning: /tmp/Rtmp8XJn7C/devtools1950f61971/shinyjs/man/hidden.Rd:49: missing file link ‘toggle’ Rd warning: /tmp/Rtmp8XJn7C/devtools1950f61971/shinyjs/man/hidden.Rd:50: missing file link ‘show’ Rd warning: /tmp/Rtmp8XJn7C/devtools1950f61971/shinyjs/man/hidden.Rd:51: missing file link ‘hide’     html                                    html       inlineCSS                               html       js                                      html       messageFuncs                            html       onevent                                 html       reset                                   html       runExample                              html       runcode                                 html       runjs                                   html       shinyjs                                 html       showLog                                 html   Rd warning: /tmp/Rtmp8XJn7C/devtools1950f61971/shinyjs/man/showLog.Rd:45: missing file link ‘logjs’     stateFuncs                              html       updateColourInput                       html       useShinyjs                              html       visibilityFuncs                         html   ** building package indices ** installing vignettes ** testing if installed package can be loaded * DONE (shinyjs) '/usr/lib64/R/bin/R' --no-site-file --no-environ --no-save --no-restore --quiet  \   CMD INSTALL '/tmp/Rtmp8XJn7C/devtools19433d7a08/colourpicker'  \   --library='/home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3' --install-tests   * installing *source* package ‘colourpicker’ ... ** package ‘colourpicker’ successfully unpacked and MD5 sums checked ** R ** inst ** preparing package for lazy loading ** help *** installing help indices Warning in file.copy(file.path(R.home("doc"), "html", "R.css"), outman) :   problem copying /usr/share/doc/R-3.3.2/html/R.css to /home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3/colourpicker/html/R.css: No such file or directory   converting help for package ‘colourpicker’     finding HTML links ... done     colourInput                             html       colourPicker                            html       colourWidget                            html       plotHelper                              html       runExample                              html       updateColourInput                       html   ** building package indices ** installing vignettes ** testing if installed package can be loaded * DONE (colourpicker) trying URL 'https://cran.rstudio.com/src/contrib/DT_0.2.tar.gz' Content type 'unknown' length 837622 bytes (817 KB) ================================================== downloaded 817 KB  Installing DT '/usr/lib64/R/bin/R' --no-site-file --no-environ --no-save --no-restore --quiet  \   CMD INSTALL '/tmp/Rtmp8XJn7C/devtools194552d18c/DT'  \   --library='/home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3' --install-tests   * installing *source* package ‘DT’ ... ** package ‘DT’ successfully unpacked and MD5 sums checked ** R ** inst ** preparing package for lazy loading ** help *** installing help indices Warning in file.copy(file.path(R.home("doc"), "html", "R.css"), outman) :   problem copying /usr/share/doc/R-3.3.2/html/R.css to /home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3/DT/html/R.css: No such file or directory   converting help for package ‘DT’     finding HTML links ... done     DT-imports                              html   Rd warning: /tmp/Rtmp8XJn7C/devtools194552d18c/DT/man/DT-imports.Rd:11: missing file link ‘%>%’     dataTableAjax                           html       dataTableOutput                         html       datatable                               html       formatCurrency                          html       proxy                                   html       replaceData                             html       styleInterval                           html       tableHeader                             html   ** building package indices ** installing vignettes ** testing if installed package can be loaded * DONE (DT) trying URL 'https://cran.rstudio.com/src/contrib/dygraphs_1.1.1.4.tar.gz' Content type 'unknown' length 303972 bytes (296 KB) ================================================== downloaded 296 KB  Installing dygraphs '/usr/lib64/R/bin/R' --no-site-file --no-environ --no-save --no-restore --quiet  \   CMD INSTALL '/tmp/Rtmp8XJn7C/devtools194d44578/dygraphs'  \   --library='/home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3' --install-tests   * installing *source* package ‘dygraphs’ ... ** package ‘dygraphs’ successfully unpacked and MD5 sums checked ** R ** inst ** tests ** preparing package for lazy loading ** help *** installing help indices Warning in file.copy(file.path(R.home("doc"), "html", "R.css"), outman) :   problem copying /usr/share/doc/R-3.3.2/html/R.css to /home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3/dygraphs/html/R.css: No such file or directory   converting help for package ‘dygraphs’     finding HTML links ... done     dyAnnotation                            html       dyAxis                                  html       dyCSS                                   html       dyCallbacks                             html       dyCandlestick                           html       dyCrosshair                             html       dyDataHandler                           html       dyDependency                            html       dyEvent                                 html       dyHighlight                             html       dyLegend                                html       dyLimit                                 html       dyOptions                               html       dyPlotter                               html       dyPlugin                                html       dyRangeSelector                         html       dyRebase                                html       dyRibbon                                html       dyRoller                                html       dySeries                                html       dySeriesData                            html       dyShading                               html       dyUnzoom                                html       dygraph-shiny                           html       dygraph                                 html       dygraphs-exports                        html   ** building package indices ** testing if installed package can be loaded * DONE (dygraphs) Skipping install of 'shinyjs' from a cran remote, the SHA1 (0.9.1) has not changed since last install.   Use `force = TRUE` to force installation trying URL 'https://cran.rstudio.com/src/contrib/shinythemes_1.1.1.tar.gz' Content type 'unknown' length 990752 bytes (967 KB) ================================================== downloaded 967 KB  Installing shinythemes '/usr/lib64/R/bin/R' --no-site-file --no-environ --no-save --no-restore --quiet  \   CMD INSTALL '/tmp/Rtmp8XJn7C/devtools1942527865/shinythemes'  \   --library='/home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3' --install-tests   * installing *source* package ‘shinythemes’ ... ** package ‘shinythemes’ successfully unpacked and MD5 sums checked ** R ** inst ** preparing package for lazy loading ** help *** installing help indices Warning in file.copy(file.path(R.home("doc"), "html", "R.css"), outman) :   problem copying /usr/share/doc/R-3.3.2/html/R.css to /home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3/shinythemes/html/R.css: No such file or directory   converting help for package ‘shinythemes’     finding HTML links ... done     shinytheme                              html       shinythemes                             html       themeSelector                           html   ** building package indices ** testing if installed package can be loaded * DONE (shinythemes) trying URL 'https://cran.rstudio.com/src/contrib/threejs_0.2.2.tar.gz' Content type 'unknown' length 447304 bytes (436 KB) ================================================== downloaded 436 KB  Installing threejs '/usr/lib64/R/bin/R' --no-site-file --no-environ --no-save --no-restore --quiet  \   CMD INSTALL '/tmp/Rtmp8XJn7C/devtools1963760808/threejs'  \   --library='/home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3' --install-tests   * installing *source* package ‘threejs’ ... ** package ‘threejs’ successfully unpacked and MD5 sums checked ** R ** data ** inst ** tests ** preparing package for lazy loading ** help *** installing help indices Warning in file.copy(file.path(R.home("doc"), "html", "R.css"), outman) :   problem copying /usr/share/doc/R-3.3.2/html/R.css to /home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3/threejs/html/R.css: No such file or directory   converting help for package ‘threejs’     finding HTML links ... done     LeMis                                   html       flights                                 html       globejs                                 html       graph2Matrix                            html       graphjs                                 html       igraph2graphjs                          html       matrix2graph                            html       scatterplot3js                          html       texture                                 html       threejs-package                         html       threejs-shiny                           html   ** building package indices ** testing if installed package can be loaded * DONE (threejs) '/usr/lib64/R/bin/R' --no-site-file --no-environ --no-save --no-restore --quiet  \   CMD INSTALL '/tmp/Rtmp8XJn7C/devtools19209227d1/shinystan'  \   --library='/home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3' --install-tests   * installing *source* package ‘shinystan’ ... ** package ‘shinystan’ successfully unpacked and MD5 sums checked ** R ** data *** moving datasets to lazyload DB ** inst ** tests ** preparing package for lazy loading ** help *** installing help indices Warning in file.copy(file.path(R.home("doc"), "html", "R.css"), outman) :   problem copying /usr/share/doc/R-3.3.2/html/R.css to /home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3/shinystan/html/R.css: No such file or directory   converting help for package ‘shinystan’     finding HTML links ... done     as.shinystan                            html   Rd warning: /tmp/Rtmp8XJn7C/devtools19209227d1/shinystan/man/as.shinystan.Rd:57: missing file link ‘model_code’ Rd warning: /tmp/Rtmp8XJn7C/devtools19209227d1/shinystan/man/as.shinystan.Rd:58: missing file link ‘model_code’ Rd warning: /tmp/Rtmp8XJn7C/devtools19209227d1/shinystan/man/as.shinystan.Rd:74: missing file link ‘get_sampler_params’     deploy_shinystan                        html   Rd warning: /tmp/Rtmp8XJn7C/devtools19209227d1/shinystan/man/deploy_shinystan.Rd:67: missing file link ‘rsconnect’     drop_parameters                         html       generate_quantity                       html       launch_shinystan                        html       launch_shinystan_demo                   html       rename_model                            html       retrieve                                html       shinystan-class                         html       shinystan-metadata                      html       shinystan-package                       html       update_sso                              html   *** copying figures ** building package indices ** installing vignettes ** testing if installed package can be loaded * DONE (shinystan) '/usr/lib64/R/bin/R' --no-site-file --no-environ --no-save --no-restore --quiet  \   CMD INSTALL '/tmp/Rtmp8XJn7C/devtools192fa8d74b/stan-dev-rstanarm-8152fb7'  \   --library='/home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3' --install-tests  \   --preclean   * installing *source* package ‘rstanarm’ ... Warning: 'cleanup' exists but is not executable -- see the 'R Installation and Administration Manual' ** libs g++ -m64 -std=c++11 -I/usr/include/R -DNDEBUG -I"`"/usr/lib64/R/bin/Rscript" --vanilla -e "cat(system.file('include', 'src', package = 'StanHeaders'))"`" -DBOOST_RESULT_OF_USE_TR1 -DBOOST_NO_DECLTYPE -DBOOST_DISABLE_ASSERTS -DEIGEN_NO_DEBUG -DBOOST_MATH_OVERFLOW_ERROR_POLICY=ignore_error -I/usr/local/include -I"/home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3/StanHeaders/include" -I"/home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3/rstan/include" -I"/usr/lib64/R/library/BH/include" -I"/home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3/Rcpp/include" -I"/usr/lib64/R/library/RcppEigen/include"   -fpic  -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches   -m64 -mtune=generic -c init.cpp -o init.o g++ -m64 -std=c++11 -shared -L/usr/lib64/R/lib -Wl,-z,relro -o rstanarm.so init.o -L/home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3/StanHeaders/lib -lStanHeaders -L/usr/lib64/R/lib -lR installing to /home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3/rstanarm/libs ** R ** data *** moving datasets to lazyload DB ** demo ** exec ** inst ** tests ** preparing package for lazy loading DIAGNOSTIC(S) FROM PARSER: Warning (non-fatal): Comments beginning with # are deprecated.  Please use // in place of # for line comments. Warning (non-fatal): Comments beginning with # are deprecated.  Please use // in place of # for line comments. Warning (non-fatal): Comments beginning with # are deprecated.  Please use // in place of # for line comments.  DIAGNOSTIC(S) FROM PARSER: Warning (non-fatal): Comments beginning with # are deprecated.  Please use // in place of # for line comments. Warning (non-fatal): Comments beginning with # are deprecated.  Please use // in place of # for line comments.  ** help No man pages found in package  ‘rstanarm’  *** installing help indices Warning in file.copy(file.path(R.home("doc"), "html", "R.css"), outman) :   problem copying /usr/share/doc/R-3.3.2/html/R.css to /home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3/rstanarm/html/R.css: No such file or directory   converting help for package ‘rstanarm’     finding HTML links ... done *** copying figures ** building package indices ** installing vignettes ** testing if installed package can be loaded Error in .doLoadActions(where, attach) :    error in load action .__A__.1 for package rstanarm: is(module, "character"): object 'm' not found Error: loading failed Execution halted ERROR: loading failed * removing ‘/home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3/rstanarm’ Installation failed: Command failed (1) > install.packages("rstanram") Installing package into ‘/home/rstudio/R/x86_64-redhat-linux-gnu-library/3.3’ (as ‘lib’ is unspecified) Warning in install.packages :   package ‘rstanram’ is not available (for R version 3.3.2)
--
 
> | >
>

--
 
  |  
 

#### Reproducible Steps:
If applicable, the steps required to reproduce the issue. If you have a reproducible example, please include it.

#### RStanARM Version:
trying to install the current version as of 8/3/2017
#### R Version:
The version of R you are running (e.g., from `getRversion()`)
> getRversion()
[1] ‘3.3.2’
#### Operating System:
Redhat Linux

**Comment by bgoodri** [comment]
```
Issuing the command install.packages("rstanarm")
produce the following error:

package ‘rstanram’ is not available (for R version 3.3.2)
```
rstanram is just misspelled. It should be `install.packages("rstanarm")`. I am not sure why `install_github` is not working in your case, but I would start with the installation from CRAN.

**Comment by gmichel-SMU** [comment]
Thanks, it is working :)
